src/copy.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs');
+const path = require('path');
+const execSync = require('child_process').execSync
+
+
+const fileToCopy = {
+  'Metadata': 'import "./Metadata.css";',
+  'ProgressBar': 'import "./ProgressBar.css";',
+  'TablePreview':  undefined,
+  'TableSchema': 'import "./TableSchema.css";',
+  'Upload': undefined
```

Comment:
don't need to use quotes inside the object keys

Reply:
that's true